app/controllers/responses_controller.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require "will_paginate/array"
```

Comment:
I don't think we should need this.

Reply:
It was needed for pagination after sorting in-place, but no longer since we're removing that logic 👍